includes/TinyMCE.php

```diff
@@ -0,0 +1,121 @@
+<?php
```

Comment:
Should we create a new namespace called Admin, add TinyMCE and Customizer ( and Admin ) to it?